### ISSUE TITLE
Disable support for DSHOT600 and DSHOT1200 for reliability reasons

### ISSUE THREAD

**Issue by digitalentity** [iNavFlight/inav#4192]
Per issue title.
Summoning @DzikuVx @fiam @stronnag @shellixyz for discussing this

**Comment by DzikuVx** [comment]
I would not go that far, but I would add a HUGE red notification in Configurator saying "You are doing this on your own responsibility and probably should not do it if you are not ready to accept the potential consequences"

**Comment by stronnag** [comment]
Agreed, and in the release notes (I added the target list already). We are in for a lot of these I think.

**Comment by teckel12** [comment]
If we disable it, users will keep demanding it to be enabled.  Better to keep it with a warning.  Then, when issues come in (which we know they will) we'll just redirect them to the warning.

With that said, it seems that most of the problems with DSHOT are on the F7.  @DzikuVx were you testing DSHOT on the F7?

**Comment by digitalentity** [comment]
Agreed. Let's make a warning in Configurator then. Moving the issue there.
